libs/coin-framework/src/derivation.ts

```diff
@@ -168,14 +168,20 @@ const modes = Object.freeze({
   polkadotbip44: {
     overridesDerivation: "44'/354'/<account>'/0'/<address>'",
   },
-  gliflegacy: {
+  glifLegacy: {
     overridesDerivation: "44'/1'/0'/0/<account>",
     tag: "legacy",
+    mandatoryEmptyAccountSkip: 5,
   },
   glif: {
-    overridesDerivation: "44'/461'/0'/0/<account>",
+    overridesDerivation: "44'/<coin_type>'/<address>'/<node>/<account>",
+    mandatoryEmptyAccountSkip: 5,
+  },
+  filecoinBIP44: {
+    overridesDerivation: "44'/<coin_type>'/<account>'/<node>/<address>",
```

Comment:
I'm not sure what you tried with `glif` and `filecoinBIP44` works? Just inverting `<address>` and `<account>`.
To be honest that derivation code is a lot of tech debt and even I am not sure of how it works in advanced cases as such.

Bottomline is, **make sure to test a lot and validate it works as you think it does, in all possible cases.**

Reply:
I have changed this in order to make it clearer. What do you think now? Now it is the same we had previously... it will now use `glif` as default derivation scheme for new accounts though.